pkg/cmd/openshift-kube-apiserver/server.go

```diff
@@ -43,22 +43,38 @@ func RunOpenShiftKubeAPIServerServer(masterConfig *configapi.MasterConfig) error
 		return kerrors.NewInvalid(configapi.Kind("MasterConfig"), "master-config.yaml", validationResults.Errors)
 	}
 
-	informers := origin.InformerAccess(nil) // use real kube-apiserver loopback client with secret token instead of that from masterConfig.MasterClients.OpenShiftLoopbackKubeConfig
-	openshiftConfig, err := origin.BuildMasterConfig(*masterConfig, informers)
-	if err != nil {
-		return err
+	bootstrappolicy.ClusterRoles = bootstrappolicy.OpenshiftClusterRoles
+	bootstrappolicy.ClusterRoleBindings = bootstrappolicy.OpenshiftClusterRoleBindings
+
+	options.AllOrderedPlugins = originadmission.CombinedAdmissionControlPlugins
```

Comment:
ugly to override global slice vars meant as constants. But I guess we have no choice.

Reply:
a carry patch might be more explicit.

Reply:
> a carry patch might be more explicit.

doing so doesn't allow running in both modes and exposes us to greater conflicts